scripts/build.sh

```diff
@@ -118,7 +118,14 @@ fi
 # Build k-NN lib and plugin through gradle tasks
 cd $work_dir
 # Gradle build is used here to replace gradle assemble due to build will also call cmake and make before generating jars
-./gradlew build --no-daemon --refresh-dependencies -x integTest -DskipTests=true -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
+./gradlew build --no-daemon --refresh-dependencies -x integTest -x test -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
```

Comment:
So `-DskipTests=true` is now `-x test`? Confirmed running?

Reply:
`-DskipTests=true` actually not skipping tests. But, I think the reason behind not changing it was if we exclude tests  then it will not build the libraries and need to build it separately.